src/uu/ln/src/ln.rs

```diff
@@ -132,7 +128,7 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         verbose: matches.get_flag(options::VERBOSE),
     };
 
-    exec(&paths[..], &settings)
+    exec(&paths, &settings)
```

Comment:
Was a bit confused when trying to understand what the [..] was doing when I was reading this, turns out its redundant